src/components/Button/PButton.vue

```diff
@@ -88,208 +78,172 @@
-  hover:text-foreground-400
+.p-button--icon-append { @apply
+  pr-2
 }
 
-.p-button--danger { @apply
-  text-white
-  bg-danger-600
-  dark:bg-danger-500
+.p-button--icon-only { @apply
+  px-2
 }
-.p-button--danger:not(.p-button--disabled) { @apply
-  hover:bg-danger-700
-  dark:hover:bg-danger-400
+
+.p-button__content { @apply
+  gap-1
+  inline-flex
+  flex-grow
+  justify-center
+  items-center
+  font-normal
 }
 
-.p-button--danger--secondary { @apply
-  bg-danger-100
-  dark:bg-danger-200
+.p-button--primary {
+  background-color: var(--p-color-button-primary-bg);
+  border-color: var(--p-color-button-primary-border);
+  color: var(--p-color-button-primary-text);
 }
-.p-button--danger--secondary:not(.p-button--disabled) { @apply
-  hover:bg-danger-200
-  dark:hover:bg-danger-100
+.p-button--primary:not(:disabled):hover {
+  background-color: var(--p-color-button-primary-bg-hover);
+  border-color: var(--p-color-button-primary-border-hover);
+  color: var(--p-color-button-primary-text-hover);
 }
-
-.p-button-xs { @apply
-  text-xs
-  px-2
-  py-1
+.p-button--primary:not(:disabled):active {
+  background-color: var(--p-color-button-primary-bg-active);
+  border-color: var(--p-color-button-primary-border-active);
+  color: var(--p-color-button-primary-text-active);
 }
 
-.p-button-xs .p-button__icon { @apply
-  h-3
-  w-3
+.p-button--danger {
+  background-color: var(--p-color-button-danger-bg);
+  border-color: var(--p-color-button-danger-border);
+  color: var(--p-color-button-danger-text);
 }
-.p-button-xs.p-button--equal-padding { @apply
-  p-2
+.p-button--danger:not(:disabled):hover {
+  background-color: var(--p-color-button-danger-bg-hover);
+  border-color: var(--p-color-button-danger-border-hover);
+  color: var(--p-color-button-danger-text-hover);
 }
-
-.p-button-sm { @apply
-  text-sm
-  px-3
-  py-1
+.p-button--danger:not(:disabled):active {
+  background-color: var(--p-color-button-danger-bg-active);
+  border-color: var(--p-color-button-danger-border-active);
+  color: var(--p-color-button-danger-text-active);
 }
 
-.p-button-sm .p-button__icon { @apply
-  h-3.5
-  w-3.5
+.p-button--primary.p-button--danger {
+  background-color: var(--p-color-button-primary-danger-bg);
+  border-color: var(--p-color-button-primary-danger-border);
+  color: var(--p-color-button-primary-danger-text);
 }
-.p-button-sm.p-button--equal-padding { @apply
-  p-2.5
+.p-button--primary.p-button--danger:not(:disabled):hover {
+  background-color: var(--p-color-button-primary-danger-bg-hover);
+  border-color: var(--p-color-button-primary-danger-border-hover);
+  color: var(--p-color-button-primary-danger-text-hover);
 }
-
-.p-button-md { @apply
-  text-base
-  px-4
-  py-2
+.p-button--primary.p-button--danger:not(:disabled):active {
+  background-color: var(--p-color-button-primary-danger-bg-active);
+  border-color: var(--p-color-button-primary-danger-border-active);
+  color: var(--p-color-button-primary-danger-text-active);
```

Comment:
We might want to come back through and make these BEM compatible at some point. But should be fine for now. Some of these selectors have a high specificity.